### ISSUE TITLE
Improvement ideas

### ISSUE THREAD

**Issue by 2E0PGS** [sudomesh/disaster-radio-website#4]
Hi nice website, a few suggestions

* A few hardware examples for the solar panels and batteries would be useful.
* A roadmap or status of feature development progress listed on the website. e.g. Maps associated issue progress https://github.com/sudomesh/disaster-radio/issues/10 (@2E0PGS can do this)
* A few hardware examples for the premade boards could be listed on the website. (@2E0PGS can do this)

**Comment by 2E0PGS** [comment]
Also a map could be added to the website displaying node locations a bit like the LoRaWAN map but less slow/laggy :P I guess it would require manual node submission via email.

**Comment by samuk** [comment]
+1 I like these ideas.

I would add a link to https://www.aliexpress.com/item/4000396836096.html that says something like; A manufacturer Lilygo offers our software on these boards, however we are not able to offer any support for the devices, and offer no guarantees to their suitability for any particular purpose. 

https://umap.openstreetmap.fr/en/map/disaster-radio_417055 could be used to create a user-generated map of nodes and embedded into a page somewhere

**Comment by paidforby** [comment]
I believe we have addressed most of the concerns outlined in this issue with recent commits and merged PRs.

I moved routing info and link from software page to a new page titled "Routing." That page also mentions "LoRa Mesh" as a name for the network created by a group of disaster radio nodes, which seems accurate. We could make a another page just for LoRa Mesh, but we would need more sensible verbiage and images to describe it. 

Maps are cool. However, disaster.radio isn't building networks, it's building technology that could be used to build networks. I would leave it up to the folks actually installing nodes to create their own maps. If we, or someone else, want to provide a tool for doing that then great. I'm not sure we are at the point of having a collaboratively built map just yet. Once we have actual hardware that someone could order and install, then it might make sense to embed a map on the site.
